hw4/task03.py

```diff
@@ -0,0 +1,8 @@
+import sys
+
+
+def my_precious_logger(text: str):
+    if text.split()[0][:5].upper() == 'ERROR':
+        print(text, file=sys.stderr)
```

Comment:
easier and more readable would be using function .startswith can be applied to any str